service/src/main/kotlin/xyz/lebkuchenfm/domain/soundboard/SoundboardService.kt

```diff
@@ -0,0 +1,27 @@
+package xyz.lebkuchenfm.domain.soundboard
+
+import com.github.michaelbull.result.Err
+import com.github.michaelbull.result.Ok
+import com.github.michaelbull.result.Result
+import xyz.lebkuchenfm.domain.eventstream.Event
+import xyz.lebkuchenfm.domain.eventstream.EventStream
+import xyz.lebkuchenfm.domain.xsounds.XSoundsService
+
+class SoundboardService(
+    private val xSoundsService: XSoundsService,
+    private val eventStream: EventStream<*>,
+) {
+    suspend fun playXSound(soundName: String): Result<Unit, PlayXSoundError> {
```

Comment:
maybe we should return xSound on success? Dunno why but I have mixed feelings about Unit here

Reply:
Unit is perfectly fine here. We just want to now that the operation succeeded, but it does not have any value associated with its success.